docs/contributor/migration-guide.md

```diff
@@ -8,78 +8,165 @@ It covers all necessary changes and deprecations to ensure a smooth transition.
-
-7. **Mandatory ModuleTemplates**
-
-   - Mandatory modules can be created using `modulectl` by setting the **.mandatory** field to `true` in the module config. If multiple ModuleTemplates for the same mandatory module get deployed to the Kyma Control Plane, Lifecycle Manager picks up the ModuleTemplate with the highest version.
-
-8. **Beta and Internal Flags**
-
-   - **Deprecated**: Beta and Internal flags are not supported for ModuleTemplates.
-   - **New**: Beta and Intenral flags are configured as part of the ModuleReleaseMeta, see [ModuleReleaseMeta Configuration](https://github.com/kyma-project/lifecycle-manager/blob/main/docs/contributor/resources/05-modulereleasemeta.md#configuration).
-
-## Migration Period
-
-**Support for Both Approaches Temporarily**: 
-
-- Both old and new approaches will be supported simultaneously by KLM during the migration period.
-- After the full migration to the new approach, KLM will no longer accept ModuleTemplate with the old naming pattern.
-- For testing, provide ModuleTemplate CRs in the new format accompanied by [ModuleReleaseMeta CRs](https://github.com/kyma-project/lifecycle-manager/blob/main/docs/contributor/resources/05-modulereleasemeta.md). KLM will attempt to use the new approach and fall back to the old approach if the new format is not found. The ModuleReleaseMeta CR can be marked as internal and beta to avoid syncing it.
-
-## Submission Process
-
-The general submission process may look as follows:
-
-1. Module Team releases a new module version in the GitHub release.
-2. Module Team configures the version to be released in the module config (in the new format) in the internal module-manifest repo.
-3. Module Team updates the version in the related channel in ModuleReleaseMeta.
-4. The submission pipeline gets triggered.
-5. After certain quality gates are passed, the new version of ModuleTemplate gets provisioned into KCP first.
-6. The updated ModuleReleaseMeta gets provisioned into KCP.
-7. Outdated ModuleTemplate (the version not mentioned in ModuleReleaseMeta) should be removed. This step allows both outdated and new versions to coexist temporarily. KLM only handles the version defined in ModuleReleaseMeta.
+```bash
+modulectl -h                # general help
+modulectl create -h         # help for 'create'
+modulectl scaffold -h       # help for 'scaffold'
+modulectl list -h           # help for 'list'
+```
+
+### 1.2 Command & Flag Differences
+
+#### 1.2.1 Command Mapping
+
+| Operation                  | Kyma CLI                 | `modulectl`                              |
+| -------------------------- | ------------------------ |------------------------------------------|
+| Scaffold module template   | `kyma module init ...`   | `modulectl scaffold ...`                 |
+| Create ModuleTemplate CR   | `kyma module create ...` | `modulectl create -c <config-file>`      |
+| List templates or versions | `kyma module list ...`   | \`modulectl list \[templates releases]\` |
```

Comment:
Why is the `modulectl list` command formatted differently? Do you mean that you can list either templates or releases?

```suggestion
| List templates or versions | `kyma module list ...`   | `modulectl list templates`<br>`modulectl list releases` |
```

Reply:
That's Right , need to be deleted as I double checked once again , there's no listing command on `modulectl`